internal/clients/embed.go

```diff
@@ -0,0 +1,54 @@
+package clients
+
+import (
+	"context"
+	"embed"
+	"errors"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+var AwsSupported = make(map[string]interface{})
```

Comment:
This should not be exported, move `IsSupported` function into this package to create a contract.